### ISSUE TITLE
[Results page] Publication information on search results page should be grey

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/orangelight#4918]
### Current display
Current color is black
![Image](https://github.com/user-attachments/assets/fdf82327-c16d-4ee6-9c72-6717222adee8)

### Desired display
Desired color is a dark grey - be sure to check this for color contrast with an accessibility checker.
![Image](https://github.com/user-attachments/assets/39c8c506-bab3-44c1-aa56-25cebb0cebe6)

Link to Figma wireframe: https://www.figma.com/design/jWuYZ9oOrTW39A7CHeHRLh/Princeton-Library?node-id=548-1289&p=f&t=0UJCBMy0tu1tvkB6-0

Hex code in figma for this grey is #717171;

### Acceptance criteria

- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

**Comment by kevinreiss** [comment]
The hex value from figma 717171 failed wcag-AAA contrast checker in Wave. It passed the other levels Of our current "gray" palette only $dark-gray 494f57 passed this level.
